edi_oca/tests/test_consumer_mixin.py

```diff
@@ -59,7 +60,7 @@ def _setup_records(cls):
 
     @classmethod
     def tearDownClass(cls):
-        cls.loader.restore_registry()
```

Comment:
:fire: this is not the same thing. Look at line 29. We still need this.

Reply:
why this has been marked as resolved? As said, this line must stay otherwise FakeModelLoader won't clean up fake models.

Reply:
maybe diff was shown wrongly but this line was restored to PR, that is why marked as resolved, anyway, it is there now